src/components/Class/ClassCard/index.js

```diff
@@ -183,60 +198,137 @@ function ClassCard({ item, editClass, enroll, style }) {
       dispatch(classActions.dropOutClass(Id));
     });
   };
+
+  const EnrolledAndTimer = () => {
+    const [Timer, setTimer] = useState(TimeLeft(item.start_time));
+    const ONE_MINUTE = 60000; //millisecs
+    setInterval(() => {
+      setTimer(TimeLeft(item.start_time));
+    }, ONE_MINUTE);
+    return (
+      <>
+        {Timer == "joinNow" ? (
+          <ExternalLink
+            style={{
+              textDecoration: "none",
+            }}
+            href={item.meet_link}
+          >
+            <Button variant="contained" fullWidth>
+              Join Now
+            </Button>
+          </ExternalLink>
+        ) : (
+          <Button disabled={true} variant="contained">
+            Starts in {Timer}
+          </Button>
+        )}
+      </>
+    );
+  };
```

Comment:
This component is used elsewhere (e.g., volunteer automation), so I think it should be put in its own file so it can be imported by other components as needed.